processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (1 of 1)
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/test.ldst2.asm (2 of 2)
--- memsim.hex	2026-09-26 17:04:26.742718876 -0400
+++ memcpu.hex	2026-09-26 17:04:26.737718927 -0400
@@ -1,7 +1,7 @@
 :04000000FEEDB3B7A7
 :04000100FFFFF4B752
 :04000200F000621395
-:0400030000000013E6
+:0400030000001DDEFE
 :0400040000000013E5
 :0400050040920233F0
 :0400060080006513FE
@@ -26,10 +26,6 @@
 :0400190000D52423C7
 :04001A000075262324
 :04001B00FFFFFFFFE5
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
 :0403C000000073378F
 :0403C1000000270110
 :0403C20000001337ED
